### ISSUE TITLE
Error with memdata stats

### ISSUE THREAD

**Issue by LostKobrakai** [tanguilp/http_cache_store_memory#1]
```
** (KeyError) key :cached_memory not found in: %{free_memory: 124829696, system_total_memory: 17179869184, total_memory: 17179869184}
    :erlang.map_get(:cached_memory, %{free_memory: 124829696, system_total_memory: 17179869184, total_memory: 17179869184})
    (http_cache_store_memory 0.3.0) /Users/benni/Temp/image_service/deps/http_cache_store_memory/src/http_cache_store_memory_stats.erl:42: :http_cache_store_memory_stats.system_memory_use/1
    (http_cache_store_memory 0.3.0) /Users/benni/Temp/image_service/deps/http_cache_store_memory/src/http_cache_store_memory_stats.erl:31: :http_cache_store_memory_stats.allocated_memory_used/1
    (http_cache_store_memory 0.3.0) /Users/benni/Temp/image_service/deps/http_cache_store_memory/src/http_cache_store_memory_lru_nuker.erl:38: :http_cache_store_memory_lru_nuker.nuke/1
    (http_cache_store_memory 0.3.0) /Users/benni/Temp/image_service/deps/http_cache_store_memory/src/http_cache_store_memory_lru_nuker.erl:31: anonymous fn/0 in :http_cache_store_memory_lru_nuker.handle_info/2
    (telemetry 1.2.1) /Users/benni/Temp/image_service/deps/telemetry/src/telemetry.erl:321: :telemetry.span/3
    (http_cache_store_memory 0.3.0) /Users/benni/Temp/image_service/deps/http_cache_store_memory/src/http_cache_store_memory_lru_nuker.erl:28: :http_cache_store_memory_lru_nuker.handle_info/2
    (stdlib 4.0.1) gen_server.erl:1120: :gen_server.try_dispatch/4
    (stdlib 4.0.1) gen_server.erl:1197: :gen_server.handle_msg/6
    (stdlib 4.0.1) proc_lib.erl:240: :proc_lib.init_p_do_apply/3
Last message: :check
State: []
```

**Comment by tanguilp** [comment]
Thanks for reporting. I made wrong assumptions about [`memsup:get_system_memory_data/0`](https://www.erlang.org/doc/man/memsup.html#get_system_memory_data-0). Will fix tomorrow.

**Comment by tanguilp** [comment]
Should be fixed in 0.3.1, feel free to report if it works as expected.

**Comment by LostKobrakai** [comment]
I can confirm this working
